authentication/current_user.go

```diff
@@ -0,0 +1,13 @@
+package authentication
+
+// CurrentUser -
+type CurrentUser struct {
```

Comment:
The way I see it, being `Current` is not something inherent from the structure, but of how it's used. That's why I'd just call it `User`.